README.md

```diff
@@ -1 +1,60 @@
-# arrow-datafusion-comet
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Apache Arrow DataFusion Comet
+
+Comet is an Apache Spark plugin that uses [Apache Arrow DataFusion](https://arrow.apache.org/datafusion/)
+as native runtime to achieve dramatic improvement in terms of query efficiency and query runtime.
```

Comment:
„dramatic“ seems a bit too dramatic😉

BTW, if it’s allowed to disclose, which companies are behind the development of comet?

Reply:
Yes it is. We should remove this word for now.

The initial contributors are from Apple (as can be seen from the PR), but we'd love to collaborate with people from the open source community who wants to achieve similar goals.